### PR TITLE
Rename "interface" struct

### PR DESCRIPTION
**Pull request by olbjo**
To avoid problems with Microsoft compilers

### CODE REVIEW

**Review by pyhys — APPROVED** [review]
Great

**Comment by fredrikdanielmoller** [comment]
Where does "dap" come from? "device access point" does not sound familiar. I searched for it in PN-AL-Protocol without success.

**Comment by olbjo** [comment]
"DAP is the access point for communication with the Ethernet interface and the processing program. "

The absence of a definition or description of DAP in the PN-AL Protocol I can not understand. Most whitepapers on Profinet refers to it. The first data slot is the DAP slot and its sub-slot provides the interface for control of device, interface and port settings. All of which I think fits nicely into this data structure.

File: src/pf_types.h [hunk 1]
```diff
@@ -2822,7 +2822,7 @@ struct pnet
          pf_lldp_name_of_device_mode_t mode;
       } name_of_device_mode;
       pf_port_t port[PNET_MAX_PORT];
-   } interface;
+   } dap;
```
Comment (hefloryd) [inline src/pf_types.h:2825]: Why not prefix with pf_ or similar? Could even be pf_interface if that is more clear than pf_dap?

Comment (olbjo) [reply src/pf_types.h]: Changed to "pf_interface".

Done

**Review by fredrikdanielmoller — APPROVED** [review]

**Pull request merged**